greenheart/simulation/technologies/steel/steel.py

```diff
@@ -768,21 +786,21 @@ def run_steel_finance_model(
         name="Lime",
         usage=feedstocks.lime_consumption,
         unit="metric tonnes of lime per metric tonne of steel",
-        cost=feedstocks.lime_unitcost,
+        cost=(feedstocks.lime_unitcost + feedstocks.lime_transport_cost),
```

Comment:
Just an FYI, that the parentheses aren't necessary from a purely code point of view, but are totally fine to leave.